arviz/tests/base_tests/test_labels.py

```diff
@@ -0,0 +1,70 @@
+from pytest_cases import fixture_ref
+from ..labels import (BaseLabeller,
+                    DimCoordLabeller,
+                    IdxLabeller,
+                    DimIdxLabeller,
+                    MapLabeller,
+                    NoVarLabeller,
+                    NoModelLabeller,
+                    mix_labellers)
+
+@pytest.fixture(scope="module")
+class Data():
+    def __init__(self):
+            self.sel= {
+                "instrument": "a",
+                "experiment": 3,
+            }
+            self.isel= {
+                "instrument": 0,
+                "experiment": 4,
+            }
+
```

Comment:
```suggestion
class Data():
    def __init__(self):
            self.sel= {
                "instrument": "a",
                "experiment": 3,
            }
            self.isel= {
                "instrument": 0,
                "experiment": 4,
            }

@pytest.fixture(scope="module")
def multidim_sels():
    return Data()
```

I think this will be the valid pytest pattern to get the fixtures to work and to do what we want them to do. Ref: [@pytest.fixture(scope="module")](https://docs.pytest.org/en/6.2.x/fixture.html#what-fixtures-are). Depending on how we organize things we can also set autouse on this one (maybe the labellers one below too): https://docs.pytest.org/en/6.2.x/fixture.html#autouse-fixtures-fixtures-you-don-t-have-to-request.